api/timerange.go

```diff
@@ -175,3 +174,18 @@ func (tr Timerange) ExtendBefore(length time.Duration) Timerange {
 func (tr Timerange) Slots() int {
 	return int((tr.end-tr.start)/tr.resolution) + 1
 }
+
+// TimeOfIndex returns the point in time corresponding to a (possibly out-of-range)
+// index. N corresponds to tr.Start + N*tr.Resolution.
```

Comment:
By N I assume you mean `point`? Might be better to rename it `index int` and match the comment + signature.